content/_help/get-started/authentication-options._es.md

```diff
@@ -21,7 +21,7 @@ Además de su contraseña, es necesario que usted configure por lo menos un mét
 **Autenticación secundaria**
 Le recomendamos que agregue dos métodos de autenticación a su cuenta. Si no puede acceder a su método de autenticación principal (por ejemplo, si pierde su teléfono), tendrá una segunda opción para acceder a su cuenta. Si usted llega a ser bloqueado o pierde su método de autenticación, Login.gov no podrá permitirle acceder a su cuenta.
 
-Si selecciona mensaje de texto o de voz, debe seleccionar un método de autenticación adicional. Login.gov no puede concederle acceso a su cuenta si se bloquea y/o pierde su método de autenticación.
+Login.gov no puede concederle acceso a su cuenta si se bloquea y/o pierde su método de autenticación.
```

Comment:
Not related to your changes, but curious why we show this text as two separate paragraphs in Spanish and French, but a single paragraph in English. Maybe we should combine them so it matches the English version.

Reply:
Yea I was wondering that, I don't know I think Danielle had updated that. I can get the translated values of the english paragraph and use that instead. MAybe it was a cost effectiveness thing?

Reply:
We could probably just combine them without re-translating, since I wouldn't expect that to have an impact on the translation.

Reply:
I dont think we can just combine, I think `Login.gov no podrá permitirle acceder a su cuenta.` is similar as the last line in this space. `Si tiene un bloqueo, tendrá que borrar su cuenta y crear una nueva.` I grabbed the translations already so gonna do that real quick.